tools/recetox_xmsannotator/recetox_xmsannotator_advanced.xml

```diff
@@ -89,21 +85,6 @@
                     cluster. Otherwise, do not account for cluster membership.
                 </help>
             </param>
-            <param name="expected_adducts" type="data" format="csv" optional="true">
-                <label>Expected adducts</label>
-                <help>
-                    Require the presence of certain adducts for a high confidence match. By default, at least the
-                    presence of an M+H adduct is required for a high confidence match.
-                </help>
-            </param>
-            <param name="boost_compounds" type="data" format="csv" optional="true">
-                <label>Validated compounds score boosting (optional)</label>
-                <help>
-                    Table of previously validated compounds to boost their scores and confidence levels.
-                    The 1st column of the table must contain IDs of compounds.
-                    The optional 2nd and 3rd columns may contain mz values and retention times.
-                </help>
-            </param>

```

Comment:
What happened with these parameters? Do we just not support boosting anymore at this point?

Reply:
Yeah, these were removed in the october version of xMSannotator.